packages/TermsAndConditions/__tests__/TermsAndConditions.spec.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { render, shallow, mount } from 'enzyme'
+
+import TermsAndConditions from '../TermsAndConditions'
+
+describe('TermsAndConditions', () => {
+  const defaultProps = {
+    copy: 'en',
+    content: ['One', 'Two', 'Three'],
+  }
+
+  const doShallow = (props = {}) => shallow(<TermsAndConditions {...defaultProps} {...props} />)
+
+  it('renders', () => {
+    const termsAndConditions = render(<TermsAndConditions {...defaultProps} />)
+    expect(termsAndConditions).toMatchSnapshot()
+  })
+
+  it('renders expanded', () => {
```

Comment:
You could add a couple tests to ensure indexed and non-indexed content render correctly.